delfin/exporter/prometheus/alert_manager.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import requests
+from oslo_config import cfg
+from oslo_log import log
+
+LOG = log.getLogger(__name__)
+CONF = cfg.CONF
+alert_mngr_opts = [
+
+    cfg.StrOpt('alert_manager_host', default='localhost',
+               help='The prometheus alert manager host'),
+    cfg.StrOpt('alert_manager_port', default='9093',
+               help='The prometheus alert manager port'),
+]
+
+CONF.register_opts(alert_mngr_opts, "PROMETHEUS_ALERT_MANAGER_EXPORTER")
+alert_cfg = CONF.PROMETHEUS_ALERT_MANAGER_EXPORTER
```

Comment:
did you check if these config works? it seems that prometheus/exporter.py  is loaded when start service.

Reply:
yes tested with different values in conf. conf changes are picked up here.